src/db.js

```diff
@@ -117,6 +117,8 @@ const currentSettlementWindowQueryReceipts = `
   GROUP BY qpPayer.fspId, qpPayer.participantId, q.currencyId, sw.createdDate
 `;
 
+const currentSettlementWindowId = 'select settlementWindowId from settlementWindow order by settlementWindowId DESC LIMIT 1;';
```

Comment:
instead of taking the top settlementwindowid, you can take the settlementwindowid of the OPEN window